examples/data-objects/badge/src/BadgeModel.tsx

```diff
@@ -7,14 +7,12 @@ import { DataObject } from "@fluidframework/aqueduct";
 import { SharedCell } from "@fluidframework/cell";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
 import { SharedMap } from "@fluidframework/map";
-import { SharedObjectSequence } from "@fluidframework/sequence";
-import { IBadgeModel, IBadgeHistory, IBadgeType } from "./Badge.types";
+import { IBadgeModel, IBadgeType } from "./Badge.types";
 import { defaultItems } from "./helpers";
 
 export class Badge extends DataObject implements IBadgeModel {
     private _currentCell: SharedCell<IBadgeType> | undefined;
     private _optionsMap: SharedMap | undefined;
```

Comment:
i wonder it this could be refactored to use maxtrix, and just have a single column and insert rows.

this would keep the functionally @tylerbutler wants

Reply:
i'd be fine to just remove this part of the change, and open an issue to move it to matrix

Reply:
I took this functionality as existing for the purpose of demonstrating `SharedObjectSequence` as a DDS to use, rather than the functionality being critical to preserve through alternate means.  Since we don't recommend `SharedObjectSequence` as a DDS to use anymore, I don't think it's particularly troubling to drop the functionality.

For implementation (`SharedObjectSequence` vs. `matrix`) I think it's not a perfect option either way, since the "source of truth" for latest state is duplicated between the cell and the history.  Without grouping the cell set and the history append, I think there's technically risk that the cell and latest history could disagree for simultaneous changes from different clients.

Reply:
@tylerbutler would be the best person to answer. he brought this up as a uses case

Reply:
Removing the history feature from the example is OK with me. We have captured the use-case, and I'm fine with a follow-up to re-add the feature if we conclude it's a good example for how we think matrix (or whatever we decide) should be used.